test/main.js

```diff
@@ -1897,18 +1899,28 @@ describe('node-fetch', () => {
 	});
 
 	it('should not timeout on cloning response without consuming one of the streams when the second packet size is less than default highWaterMark', function () {
+		// TODO: fix test.
+		if (!isNodeLowerThan('v16.0.0')) {
+			this.skip();
+		}
```

Comment:
do we need to create a follow up issue for the free `// TODO:` comments?

Reply:
That is a Good idea

Reply:
#1210